developer/src/kmcmplib/src/NamedCodeConstants.cpp

```diff
@@ -264,14 +264,14 @@ int NamedCodeConstants::GetCode_IncludedCodes(const KMX_WCHAR *codename)
 
 const int
  HangulSBase = 0xAC00,
- HangulLBase = 0x1100,
- HangulVBase = 0x1161,
- HangulTBase = 0x11A7,
+ //HangulLBase = 0x1100,
+ //HangulVBase = 0x1161,
+ //HangulTBase = 0x11A7,
  HangulLCount = 19,
  HangulVCount = 21,
- HangulTCount = 28,
- HangulNCount = HangulVCount * HangulTCount,   // 588
- HangulSCount = HangulLCount * HangulNCount;   // 11172
+ HangulTCount = 28;
+ //HangulNCount = HangulVCount * HangulTCount;   // 588
+ //HangulSCount = HangulLCount * HangulNCount;   // 11172
```

Comment:
I'm  not sure what is going on here with the and above

Reply:
These are unused constants, but I didn't want to delete them because they are helpful references.